README.md

```diff
@@ -1,2 +1,30 @@
-# k8s-node-net-conf [WIP]
-This is in initial phase of implementation. See ```docs/README.md``` for overall design and functionality.
+# The "Why"
+With hybrid clouds, host networking setup is even becoming more challenging. Different payloads have different networking requirements, and not everything could be satisfied as overlays on top of the main interface of the host (SR-IOV, L2, ...).
+The [CNI](https://github.com/containernetworking/cni) standard enables different solutions for connecting networks on the host with pods. Some of them are [part of the standard](https://github.com/containernetworking/cni), and there are some that support: [OVS bridges](https://github.com/containernetworking/cni), [SR-IOV](https://github.com/containernetworking/cni), and more...
+However, in all of these cases, the host must have the networks setup before the pod is scheduled. Setting up the networks in a dynamic and heterogenous cluster, with dynamic networking requirements, is a challenge by itself - and this is what this project is addressing.
+# The "How"
+We use [nmstate](https://nmstate.github.io/) to perform state driven network configuration on each host, as well as to report back its current state. 
+The system defines 2 CRDs ([Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)): ```NodeNetworkState``` and ```NodeNetConfPolicy```. 
```

Comment:
I think you should point here to Eddy's CRD document

Reply:
not an upstream doc. will do that once it is in git.

Reply:
I don't understand your reply. https://docs.google.com/document/d/1282BcYjYGIIxQKgMYi3nQodB4ML_gw9BSs5AXB7QUtg/edit is upstream, and it is the thing that defines the CRDs that you mention.

Reply:
added the reference